aten/src/THC/THCTensorRandom.cu

```diff
@@ -62,19 +64,32 @@ __global__ void set_rngstate_kernel(curandStateMtgp32 *state, mtgp32_kernel_para
 
 __host__ void THCRandom_setRNGState(THCState* state, THByteTensor *rng_state)
 {
-  Generator* gen = THCRandom_getGenerator(state);
+  THCGenerator* gen = THCRandom_getGenerator(state);
 
   static const size_t states_size = MAX_NUM_BLOCKS * sizeof(curandStateMtgp32);
   static const size_t seed_size = sizeof(gen->initial_seed);
-  static const size_t total_size = states_size + seed_size;
-  THArgCheck(THByteTensor_nElement(rng_state) == total_size, 1, "RNG state is wrong size");
+  static const size_t offset_size = sizeof(gen->philox_seed_offset);
+  static const size_t total_size = states_size + seed_size + offset_size;
+  bool deprecated = false;
```

Comment:
Well it's not really deprecated. It just doesn't have the philox seed.

Reply:
Got it. Is `no_philox_seed` better?

Reply:
Yeah that would be nicer